back-end/tests/models/assignedBadges.test.js

```diff
@@ -0,0 +1,31 @@
+const {
+  createAssignedBadges,
+  createAdventurer,
+  createBadge,
+} = require("../seeds");
+const AssignedBadges = require("../../db/models/classProgress");
```

Comment:
All tests in that file are actually failing 😱 
That's because you're not importing from the right model file here 😉

Reply:
# derp